### DISCUSSION TITLE
Which inline completion item is showing ?

### DISCUSSION THREAD

**Discussion by qqaimh** [microsoft/vscode-discussions#1613] [Extension Development QnA]
ProvideInlineCompletionItems can return an array of vscode.InlineCompletionItem, we can use Alt + [ or Alt + ] to show  before or next inline suggestion item. But how can we konw which inline completion item is showing ?

**Comment by isidorn** [comment]
@hediet might be able to help here

**Reply by robconery** [reply]
Quick ping to @hediet to see if you had a thought on this?
